spec/book_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../book'
+
+describe Book do
+  before(:each) do
+    @book = Book.new('publisher', 'good', 1, Date.new(2002, 3, 2).to_s)
+  end
+
+  context '#move_to_archive' do
+    it "should return true if cover_state equals 'bad', even if parent's method returns false" do
+      bad_book = Book.new('publisher', 'bad', 1, Date.new(2015, 2, 3).to_s)
+      expect(bad_book.move_to_archive).to be true
+    end
+
+    it "should return true if the parent's method returns true" do
```

Comment:
I think you forget to write the correct description of the test as it should be about function `move_to_archive` not `parent` . Please fix it.